projects/Mallard/src/authentication/authorizers/IdentityAuthorizer.ts

```diff
@@ -56,11 +74,23 @@ const authWithTokens = async (
     )
 }
 
-const getUserNameFromParams = (params: AuthParams) => {
-    if ('email' in params) return params.email
-    if ('facebook-access-token' in params) return 'gu-editions::token::facebook'
-    if ('google-access-token' in params) return 'gu-editions::token::google'
+// TODO: This function needs a unit test
+const getUserName = (authType: AuthType, params: AuthParams) => {
```

Comment:
this is currently failing the validate step (my fault as I wrote it!) - here's what I did to fix it https://github.com/guardian/editions/commit/94ffa33af3dc24b40fa5e3b6ce3dd07460b81dcd